documentation/assemblies/modules

```diff
@@ -0,0 +1 @@
+/home/pmellor/Documentation/strimzi-kafka-operator/documentation/modules
```

Comment:
I guess this is not supposed to be here with this exact path. Or?

Reply:
:disappointed: Sorry. Didn't show up on my local build.